### ISSUE TITLE
Slow interface

### ISSUE THREAD

**Issue by Thetruecesar** [saVantCZ/PS5reborn#22]
Thanks for continuing to update this great theme, but since version 1.2.0 or 1.3.0 the interface became slower for me and when navigating between games in the library or on the home screen the game images get stuck.
I have a good cpu and gpu, why could this be happening?
Thanks

**Comment by saVantCZ** [comment]
Can you try with disabled success story plugin?

**Comment by Thetruecesar** [comment]
Yes I also disabled it, but the same thing keeps happening, the theme slows down

**Comment by saVantCZ** [comment]
Can you make video?

**Comment by foss-lover34** [comment]
> Thanks for continuing to update this great theme, but since version 1.2.0 or 1.3.0 the interface became slower for me and when navigating between games in the library or on the home screen the game images get stuck. I have a good cpu and gpu, why could this be happening? Thanks

you probably have some settings messed up i dont think its this theme fault, try resetting playnite

**Comment by saVantCZ** [comment]
Success story plugin or Steam Screenshots plugin "can" freeze browsing games, but only one time until it creates cache... once cache is created, it will never lag
